### PR TITLE
watson: 1.5.2 -> 1.7.0

### PR DESCRIPTION
**Pull request by nathyong**
###### Motivation for this change

Update to the latest version on PyPI.

Current nixos-version:
```$ nixos-version
19.03.172927.30a82bba734 (Koi)
```

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review 63972"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: maintainers/maintainer-list.nix
Comment (matthewbauer) [inline maintainers/maintainer-list.nix]: missing semicolon

Comment (nathyong) [reply maintainers/maintainer-list.nix]: Addressed - thanks

**Comment by worldofpeace** [comment]
Hi @nathyong. Most contributions like this should target the `master` branch.
If you'd like to update a package on the stable branch you should [backport](https://nixos.org/nixpkgs/manual/#submitting-changes-stable-release-branches) this after it has been merged to master.

**Comment by nathyong** [comment]
Hey @worldofpeace I tried packaging and testing on the master branch, but I encountered some errors while running `nix-review` while building another derivation. I can rebase this onto master, though, no worries.

**Comment by worldofpeace** [comment]
The attribute of this package is `watson` so the commit msg should reflect that.

**Comment by worldofpeace** [comment]
The error you mentioned with `nix-review` is because a dependency `pytest-mock` tests are failing on master.

**Comment by nathyong** [comment]
I assume that the merge will be blocked until those are fixed, then.

Just to ensure that I've understood it correctly: The package is located in `pkgs/applications/office/watson`, and it is declared in nixpkgs at `pkgs/top-level/all-packages.nix:20820` under the name `watson`. The name `td-watson` is merely a PyPI detail to build this package.

**Comment by worldofpeace** [comment]
> I assume that the merge will be blocked until those are fixed, then.

I can bypass that locally by disabling the `checkPhase` of `pytest-mock`. 
Then I can  review this properly `watson`.

> Just to ensure that I've understood it correctly: The package is located in pkgs/applications/office/watson, and it is declared in nixpkgs at pkgs/top-level/all-packages.nix:20820 under the name watson. The name td-watson is merely a PyPI detail to build this package.

Yes that's correct. However I usually advise people to normalize `pname` to something expected here, `watson`, and then give `fetchPypi` `td-watson`.

**Review by worldofpeace — CHANGES_REQUESTED** [review]
Can you please drop `doCheck = true;` (some reason can't suggest that on GitHub) and use the new `pytest` command in the `checkPhase`?

Other than that LGTM.

**Review by worldofpeace — APPROVED** [review]

**Pull request merged**
